src/useInputState.js

```diff
@@ -0,0 +1,13 @@
+import { useState } from "react";
+
+export default () => {
+  const [value, setValue] = useState("");
+
+  return {
+    value,
+    onChange: (event) => {
+      setValue(event.target.value);
+    },
+    reset: () => setValue(""),
+  };
+};
```

Comment:
This can be written inside the component. There's no need for the extra hook!

Q: Do you know what the React hooks do? (in this case, `useState`)
Q2: Do you know that you're building a custom hook and might be harder to test the code?

Reply:
PS: we're going to have a unit testing checkpoint!